### ISSUE TITLE
Laravel 9 Support

### ISSUE THREAD

**Issue by ludgerey** [SynergiTech/laravel-postal#30]
Composer install fails for `laravel-postal` with Laravel 9. Laravel 9 is released as stable since February 8th, 2022.

```
composer update                                                                                         
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - synergitech/laravel-postal[v3.3.0, ..., v3.3.2] require laravel/framework ^6.0|7.0|^8.0 -> found laravel/framework[v6.0.0, ..., 6.x-dev, v7.0.0, v8.0.0, ..., 8.x-dev] but it conflicts with your root composer.json require (^9.0).
    - Root composer.json requires synergitech/laravel-postal ^3.3 -> satisfiable by synergitech/laravel-postal[v3.3.0, v3.3.1, v3.3.2].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

Laravel 9 support might need some changes to this library as the new switched from SwiftMailer to SymfonyMailer which similar but requires some minor changes (see https://laravel.com/docs/9.x/upgrade#symfony-mailer)

**Comment by willpower232** [comment]
You can see I've prepared most of the work for this in #29 but we had to wait for a laravel patch release for a bug that was introduced and we won't get an opportunity to review this internally until next week now.

**Comment by Josh-G** [comment]
Released in [v4.0.0](https://github.com/SynergiTech/laravel-postal/releases/tag/v4.0.0)
